WiseBuy/Managers/DatabaseManager.m

```diff
@@ -46,8 +63,8 @@ +(void)registerUser:(User *)user withCompletion:(void(^)(BOOL success, NSError *
 + (void)logoutUser:(UIViewController *)vc {
     SceneDelegate *sceneDelegate = (SceneDelegate *) vc.view.window.windowScene.delegate;
     UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-    UINavigationController *initialNavigationController = [storyboard instantiateViewControllerWithIdentifier:@"InitialNavigationController"];
-    sceneDelegate.window.rootViewController = initialNavigationController;
+    UIViewController *initialViewController = [storyboard instantiateViewControllerWithIdentifier:kLoginViewController];
+    sceneDelegate.window.rootViewController = initialViewController;
     
```

Comment:
Same q as other PRs - why was this changed from UINavigationController to UIViewController. Seems unrelated to the changes mentioned in the description